src/reducers/auth.js

```diff
@@ -2,7 +2,7 @@ import * as types from '../actions';
 
 const initialState = {
   loading: false,
-  authorized: window.localStorage.authorized || false
+  authorized: window.localStorage.authorized || null
```

Comment:
This was updated to `null`, but [this line](https://github.com/coralproject/cay/pull/59/files#diff-ee8b9b1575b213869cc58e7ee67c8d9bR73) still checks for `this.props.authorized === false` which will fail.